osu.Game/Graphics/UserInterface/DrawableStatefulMenuItem.cs

```diff
@@ -63,7 +63,7 @@ private void updateState(ValueChangedEvent<object> state)
                     stateIcon.Alpha = 0;
                 else
                 {
-                    stateIcon.Alpha = 1;
+                    stateIcon.Alpha = state.NewValue != state.OldValue ? 0 : 1;
```

Comment:
This should just be done by completely unbinding `updateState` from being called on state change, but instead be called once where it was on `LoadComplete()`.

Reply:
In this case, wouldn't unbinding `updateState` also make it so that menus that aren't closed on state changes from displaying the correct value? Or am I missing something?

Reply:
Correct, I'm talking about both things separately, you would want the `updateState` to be bound if the menu doesn't close on click.